metricbeat/module/kubernetes/_meta/docs.asciidoc

```diff
@@ -146,7 +146,7 @@ roleRef:
 === Compatibility
 
 The Kubernetes module is tested with the following versions of Kubernetes:
-1.28.x, 1.29.x, 1.30.x and 1.31.x
+1.28.x, 1.29.x, 1.30.x, 1.31.x and 1.32.x.
```

Comment:
Not a big deal, but we usually removed the versions that were no longer being supported, so that would be everything before 1.30.

See end of life: https://endoflife.date/kubernetes

Reply:
Ack. Versions as old as v1.25 are in extended support in [AWS EKS](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#available-versions-extended) so I prefer to keep the older versions here.